_episodes/01-basics.md

```diff
@@ -28,41 +28,40 @@ Word's [Track Changes](https://support.office.com/en-us/article/Track-changes-in
 history](https://support.google.com/docs/answer/190843?hl=en), or LibreOffice's [Recording and Displaying Changes](https://help.libreoffice.org/Common/Recording_and_Displaying_Changes).
 
 Version control systems start with a base version of the document and
-then save just the changes you made at each step of the way. You can
-think of it as a tape: if you rewind the tape and start at the base
-document, then you can play back each change and end up with your
-latest version.
+then record changes you make each step of the way. You can
+think of it as a recording of your progress: you can rewind to start at the base
+document, play back each change you made, eventually arriving at your
+more recent version.
 
 ![Changes Are Saved Sequentially](../fig/play-changes.svg)
 
 Once you think of changes as separate from the document itself, you
-can then think about "playing back" different sets of changes onto the
-base document and getting different versions of the document. For
-example, two users can make independent sets of changes based on the
-same document.
+can then think about "playing back" different sets of changes, resulting in 
+different versions of the document. For example, two users can make independent
+sets of changes on the same document. 
```

Comment:
This explanation is slightly more concise, but I think removing another mention of the "base document" might be a missed opportunity to reinforce the desired mental model. I would stick that back in and keep the rest of your changes.

Reply:
Again, agree with the comment from @somexpert above. Should be a very minor change to the PR.

Reply:
Sounds good